openapi-specs/cspm/AssetMicroService.json

```diff
@@ -71,11 +71,11 @@
     "/uai/v1/asset": {
       "post": {
         "tags": [
-          "Asset Explorer"
+          "Asset Domain Service"
         ],
         "summary": "Get Asset",
-        "description": "Returns detailed information for the asset with the given id.\n\nThe data field in the response object contains the raw JSON blob as returned by the source cloud service provider API for the given asset.\n\nOnly the rrn parameter in the request body is used for this API. Ignore the timelineItemId and findingType fields.\n\n Ensure to add the required parameters for the asset query parameter with type as follows: \n\n | **Type** | **Conditionally Required Query Parameters**                                              |\n| ------------------------------- | ----------------------------------------------------------------- |\n| external_finding                | <ul><li>findingType</li><li>riskFactors </li></ul>                                   |\n| alerts                          | alertIds                                                          |\n| attack_path                      | attackPathIds                                                     |\n| package_info                    | <ul><li>vulnerabilityInfoTypeId</li><li>filters</li></ul>                           |\n| labels                          | filters                                                           |\n| vulnerability_aggregates        | filters                                                           |\n| process_info                    | filters                                                          |\n| vulnerabilities_group_by_type   |<ul><li>vulnerabilityInfoTypeId</li><li>   filters</li></ul>                           |\n| asset_cwp_vulns                 |<ul><li>vulnerabilityInfoTypeId</li><li>vulnerabilityInfoType</li><li>filters</li></ul> |\n\n",
```

Comment:
@AbiMano4688 - not sure if this was something you manually added to the description. It was not included in the latest spec file from Asset Micro Service.

Reply:
https://redlock.atlassian.net/browse/RLP-128520

Reply:
The AssetMicroservice.json file contains the changes that are reviewed and published in pan.dev. Apart from the new updates, retain all the existing info in the doc repo and if possible include the difference to the engineering repo. At times, the engineering team fails to update the spec file with the changes communicated to them.